app/scripts/modules/core/pipeline/config/stages/findImageFromTags/findImageFromTagsStage.html

```diff
@@ -0,0 +1,12 @@
+<div class="form-horizontal">
+  <stage-config-field label="Package" help-key="pipeline.config.findImageFromTags.packageName">
```

Comment:
Actually opt'd for no help-key ... will revise if it proves confusing to people.